google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/GoogleAccountPluginUninstaller.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.startup;
+
+import com.google.cloud.tools.intellij.analytics.UsageTrackerProvider;
+import com.google.cloud.tools.intellij.util.GctBundle;
+import com.google.cloud.tools.intellij.util.GctTracking;
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.ide.plugins.PluginManagerConfigurable;
+import com.intellij.ide.plugins.PluginManagerUISettings;
+import com.intellij.ide.plugins.UninstallPluginAction;
+import com.intellij.openapi.application.ex.ApplicationManagerEx;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.extensions.PluginId;
+import com.intellij.openapi.ui.MessageDialogBuilder;
+
+/**
+ * Google Login used to be implemented as its own plugin that was distributed separately, but
+ * depended on by the Cloud Tools plugin.  We have now incorporated login directly into the
+ * Cloud Tools plugin and this class checks whether the legacy Google Account plugin is still
+ * installed and removes it, prompting the user to restart.
+ *
+ * Keeping the old Google Account plugin installed causes problems such as duplicating the login
```

Comment:
nit: Use a `<p>` tag at the beginning of this statement so it's in a separate paragraph.

Reply:
Oops sorry I didn't specify, but it should look like this:

    /**
     * <p>Keeping the old...
     * ...wrap like normal...
     */